packages/react-components/react-spinner/etc/react-spinner.api.md

```diff
@@ -39,11 +39,12 @@ export type SpinnerProps = Omit<ComponentProps<SpinnerSlots>, 'size'> & {
 export type SpinnerSlots = {
     root: NonNullable<Slot<'div'>>;
     spinner?: Slot<'span'>;
+    spinnerTail?: NonNullable<Slot<'span'>>;
     label?: Slot<typeof Label>;
 };
 
 // @public
-export type SpinnerState = ComponentState<SpinnerSlots> & Required<Pick<SpinnerProps, 'appearance' | 'delay' | 'labelPosition' | 'size'>> & {
+export type SpinnerState = ComponentState<Required<SpinnerSlots>> & Required<Pick<SpinnerProps, 'appearance' | 'delay' | 'labelPosition' | 'size'>> & {
```

Comment:
We're making the slots required on state now. Is that a breaking change?

Reply:
Reverted